src/Telemetry/Admin_Resources.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace StellarWP\Telemetry;
+
+class Admin_Resources {
+
+	const SCRIPT_HANDLE  = 'stellarwp-telemetry-admin';
+	const SCRIPT_VERSION = '1.0.0';
+
+	public function enqueue() {
+		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_scripts' ] );
+		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_styles' ] );
+	}
+
+	public function enqueue_scripts() {
+		$script_path = apply_filters( 'stellarwp/telemetry/script_path', $this->get_asset_path() . 'resources/js/scripts.js' );
+
+		wp_enqueue_script(
+			self::SCRIPT_HANDLE,
+			$script_path,
+			[ 'jquery' ],
+			self::SCRIPT_VERSION,
+			true
+		);
+	}
+
+	public function enqueue_styles() {
+		$style_path = apply_filters( 'stellarwp/telemetry/style_path', $this->get_asset_path() . 'resources/css/styles.css' );
+
+		wp_enqueue_style(
+			self::SCRIPT_HANDLE,
+			$style_path,
+			[],
+			self::SCRIPT_VERSION
+		);
```

Comment:
Hey @dotjon, @ChrisMKindred and I talked this over and it might be better to document how plugins should override these styles and scripts. They can define the `self::SCRIPT_HANDLE` as a requirement for whatever their override file is and it'll be loaded only if these scripts are available.
```php
wp_enqueue_style( 'my_override_handle', 'path/to/opt-in-overrides.css', [ 'stellarwp-telemetry-admin' ], '1.0.0' );
```

Reply:
We'll also want to add in some checks to verify that we're on a page that uses these assets so they're not loaded on every page of the admin.